src/components/data/DataTable.tsx

```diff
@@ -59,6 +65,25 @@ type ComponentProps<Row> = {
    */
   onConfirmRow?: (r: Row) => void;
 
+  /** Currently active order */
+  order?: Order<keyof Row>;
+
+  /**
+   * Callback when active order changes, by clicking a column header.
```

Comment:
```suggestion
   * Callback invoked when user clicks a column header to change the sort order.
```

There might be other triggers that change the sort order in future, but I think it makes sense to describe this in terms of the main trigger.